lib/spack/docs/getting_started.rst

```diff
@@ -926,75 +926,33 @@ Once ``curl`` has been installed, you can similarly install the others.
 Environment Modules
 """""""""""""""""""
 
-In order to use Spack's generated environment modules, you must have
-installed one of *Environment Modules* or *Lmod*.  On many Linux
-distributions, this can be installed from the vendor's repository.  For
-example: ``yum install environment-modules`` (Fedora/RHEL/CentOS). If
-your Linux distribution does not have Environment Modules, Spack can
-build it for you!
+In order to use Spack's generated module files, you must have
+installed one of *Environment Modules* or *Lmod*. The preferred method
```

Comment:
Is "one of" really necessary?

Reply:
I didn't touch this sentence, but I agree - we can remove that!